### DISCUSSION TITLE
Dicebear API Error 502 Bad Gateway

### DISCUSSION THREAD

**Discussion by riyantowibowo** [dicebear/dicebear#416] [Q&A] [answered]
Some seed got error 502 bad gateway response, this is an example api link that I get from the playground site :

When I make lowercase this seed value
https://api.dicebear.com/8.x/adventurer/svg?seed=Bandit
to
https://api.dicebear.com/8.x/adventurer/svg?seed=bandit

It said  error 502 bad gateway

![Screenshot 2024-06-09 104613](https://github.com/dicebear/dicebear/assets/7629714/8b936577-5b63-4ad0-91eb-bbe1e088256a)

**Comment by FlorianKoerner** [comment, marked as answer]
Hi and thanks for bringing this to my attention!

The 502 Bad Gateway error should be fixed now. Is it working as expected for you now?

**Comment by riyantowibowo** [comment]
Thanks, it works now
